app/components/Poll/Poll.css

```diff
@@ -1,185 +1,167 @@
+  + .voteButton {
+    margin-left: 2px;
   }
 }
 
-.pollHeader {
-  border-radius: 8px;
-  margin-bottom: 20px;
-  margin-left: 20px;
-  font-size: 16px;
-  color: var(--lego-font-color);
-}
-
-.voteButton {
-  background: var(--lego-red-color);
-  color: var(--lego-color-gray-light);
-  border: 1px solid var(--border-gray);
+.voteOptions {
   width: 100%;
-  margin: 0 !important;
-  font-size: 15px;
-  max-width: 400px;
-}
-
-.voteButton:hover {
-  opacity: 0.8;
+  padding-top: 35px;
 }
 
-html[data-theme='dark'] .voteButton {
-  color: var(--color-dark-gray-3);
+.bottomInfoWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
-.moreOptionsLink {
-  justify-content: space-between;
+.totalVotesInfo {
+  display: flex;
 }
 
-.arrow {
-  margin-top: 9px;
-  cursor: pointer;
-
-  &:hover {
-    transform: scale(1.5);
-    color: var(--color-red-3);
-    transition: transform 0.2s;
-  }
+.resultsHiddenInfo {
+  font-style: italic;
 }
 
-.blurContainer {
-  display: none;
-  position: absolute;
-  justify-content: center;
+.pollTable {
   width: 100%;
-  height: 100%;
+  font-size: 14px;
 }
 
-.blurOverlay {
-  position: absolute;
-  z-index: 2;
-  color: var(--color-black);
-  margin-top: 25px;
+.pollTable tr {
+  width: 100%;
 }
 
-.optionWrapper:hover .blurContainer {
-  display: flex;
+.pollTable td {
+  border: 0;
+  padding: 5px;
 }
 
-.optionWrapper:hover .blurEffect {
-  filter: blur(3px);
-  pointer-events: none;
+.pollTable .textColumn {
+  text-align: right;
+  line-height: 16px;
+  word-wrap: break-word;
+  width: fit-content;
+  max-width: 200px;
+  padding-right: 13px;
+  border-right: 3px solid #c5c5c5;
 }
 
-.blurArrow {
-  margin-top: 40px;
+.pollTable .graphColumn {
+  width: auto;
+  min-width: 150px;
+  padding-left: 13px;
+  padding-right: 15px;
 }
 
-.alignItems {
+.fullGraph {
   display: flex;
-  justify-content: center;
+  background-color: var(--color-mono-gray-5);
+  color: #000;
+  word-wrap: break-word;
+  width: 100%;
 }
 
-.answered {
-  margin: 15px 0;
-  text-align: center;
-  font-weight: bold;
+.pollGraph {
+  background-color: var(--lego-red);
+  font-style: italic;
+  font-weight: 300;
+  color: var(--color-white);
+  height: 30px;
+  padding-left: 8px;
+  animation: graph 1.2s cubic-bezier(0.41, 0.8, 0.4, 0.94);
+  border-radius: 0 2px 2px 0;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 25%);
 }
 
-.bottomInfo {
-  display: flex;
-  justify-content: space-between;
+.bottomBar {
+  width: 100%;
+  height: 70px;
+  background-color: var(--lego-red);
 }
 
-.resultsHidden {
-  font-style: italic;
+.arrowUp,
+.arrowDown {
+  text-shadow: 0 2px 2px rgba(0, 0, 0, 25%);
+  transform: scale(250%);
+  color: var(--lego-card-color);
+  transition: 0.8s margin-bottom;
 }
 
-@media (--mobile-device) {
-  .blurContainer {
-    display: flex;
-  }
+.arrowUp {
+  margin-bottom: 25px;
+}
 
-  .blurEffect {
-    filter: blur(3px);
-    pointer-events: none;
-  }
+.arrowDown {
+  margin-bottom: 10px;
```

Comment:
I _really_ don't like the way you're changing the margin. The way the icon slightly moves is very ugly, as it makes the component look "non-robust" or sort of "hacky" - difficult to explain.